### DISCUSSION TITLE
Always show all client's monitors (on multi monitor systems) for selection

### DISCUSSION THREAD

**Discussion by paulsommer** [rustdesk/rustdesk#4821] [Feature Request]
I remote support some clients who have two or three monitors. Since Windows tends to open each new window on the "other" screen, I always have to click on the monitor icon first and then choose the monitor I want to switch to. If it's the wrong one I have to do the two clicks again. 

I would like RustDesk to always show all available monitors directly on the control window to make switching between monitors easier.
The little RustDesk control window should have anough space for that..

I would also appreciate it if the active monitor was highlighted.

Thank you for taking note.

P.S. I'm using rustdesk on Linux (self hosted)

**Comment by SergeyMy** [comment]
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/5ecdb18e-7278-4e88-85b6-1eeac4625f27)
so what is it?

**Comment by paulsommer** [comment]
That's exactly what I was thinking of.

Is this a photoshopped image?

**Reply by deep-soft** [reply]
You have to enable in Settings:
![image](https://github.com/rustdesk/rustdesk/assets/10506197/3d7b8e85-fa4e-42f1-8a79-d0a4456400c7)

**Comment by paulsommer** [comment]
Oh, silly me ;-)
Thanks a lot.
